### DISCUSSION TITLE
AMP validation error: Tag 'html' marked with attribute 'data-ampdevmode'. Validator will suppress errors regarding any other tag with this attribute.

### DISCUSSION THREAD

**Discussion by cheong12001** [vercel/next.js#14326] [Help]
I have develop  a single amp page application and deploy with production environment, however, when i browser my website, the amp validator shows "Tag 'html' marked with attribute 'data-ampdevmode'. Validator will suppress errors regarding any other tag with this attribute." I don't know how to due with it. Thank you in advanced

**Comment by ijjk** [comment]
Hi, it sounds like you are running your application in dev mode as the `data-ampdevmode` attribute is only added in dev mode and not in production mode in Next.js. You need to make sure you are using `next build` and `next start` to deploy for production if you are not deploying using a streamlined workflow like [Vercel](https://vercel.com) which removes the need to take of this yourself

**Reply by MunnyReol** [reply]
Thanks, its work.

**Reply by dotsinspace** [reply]
Still getting this error on development server tried vercel dev even. Please help me out.

**Reply by MunnyReol** [reply]
> Still getting this error on development server tried vercel dev even. Please help me out.

It's work in production mod
